sensu/standalone_test.go

```diff
@@ -0,0 +1,116 @@
+package sensu
+
+import (
+	"testing"
+
+	stdCheck "github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/upfluence/sensu-go/sensu/check"
+	stdClient "github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/upfluence/sensu-go/sensu/client"
+)
+
+func TestMissingCommandKey(t *testing.T) {
+	standaloneProcessor := &Standalone{check: &stdCheck.Check{}}
+
+	err := standaloneProcessor.execute()
+
+	if err != commandKeyError {
+		t.Fatalf("Expected error to be \"%s\" but got \"%s\" instead!",
```

Comment:
When you write a multi-line method call, or a multiline struct definition. Could you: 

* Add a new line after the opening sign ( `(` for functions, `{`  for structs )
* Add a new line before the closing sign ( `)` for functions, `}`  for structs )


Here is an example for the function:

```golang
t.Fatalf(
    "Expected error to be \"%s\" but got \"%s\" instead!",
    commandKeyError,
    err,
)
```

Here is an example for a struct definition:

```golang
t.publishParameters = &transportPublishParameters{
    exchangeType,
    exchangeName,
    key,
    message,
}
```